### PR TITLE
JavaScript Capstone: Foodify Kitchenball

### PR DESCRIPTION
**Pull request by HammadSiddique**
### Hi there 👋
Me (@HammadSiddique)  and my partner (@ibr5500) created this javascript capstone together using [TheMealDB](https://www.themealdb.com/api.php) API.

## This project is built on the following objectives.

- Use JavaScript to make websites dynamic and build basic single page apps.
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use webpack.
- Apply JavaScript best practices and language style guides in code.
- Use AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code synchronous.
- Perform a code review for a team member.

### CODE REVIEW

File: src/modules/comment.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import commentsCounter from './commentCounter.js';
+
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
```
Comment (Cathella) [inline src/modules/comment.js:3]: ✅ The project should use the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments and reservations). 👍🏻

File: src/modules/displayItems.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import displayPopup from './popupWindow.js';
+import { getLikes, likeInteract } from './like.js';
+import itemsCounter from './itemCounter.js';
+
+const recipeCount = document.querySelector('.item-count');
+const recipeCards = document.querySelector('.recipe-cards');
+
+const displayCards = async () => {
+  fetch('https://www.themealdb.com/api/json/v1/1/filter.php?c=Vegetarian')
+    .then((response) => response.json())
```
Comment (Cathella) [inline src/modules/displayItems.js:10]: ✅ The web app uses an API selected by students that base its development around it. 👍🏻

File: src/modules/displayItems.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    .then((data) => {
+      let html = '';
+      data.meals.forEach((meal) => {
+        html += `<div class="card" id="${meal.idMeal}">
+               <div class="card-img">
+                   <img src="${meal.strMealThumb}" alt="${meal.strMeal}">
+               </div>
+               <div class="card-info">
+                   <h2>${meal.strMeal}</h2>
+                   <ul>
```
Comment (Cathella) [inline src/modules/displayItems.js:20]: ✅ When the page loads, the web app should retrieve data from the selected API and show the list of items on the screen. 👍🏻

File: src/modules/displayItems.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                       <li><button class="like-btn"><i class="fa-regular fa-heart"></i><span class="like-count">0</span></button></li>
+                   </ul>
+               </div>
+             </div>`;
+        recipeCards.innerHTML = html;
+      });
+      displayPopup();
+      getLikes();
+      likeInteract();
+      itemsCounter(data.meals, recipeCount);
```
Comment (Cathella) [inline src/modules/displayItems.js:31]: ✅ Counters should be present in all the interfaces that show the number of items 👍🏻

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @HammadSiddique, @ibr5500 👋🏻, 

Your app looks beautiful 😍, with a creative header to match the theme. Every requirement for this project has been fulfilled. You have done an amazing job as a team 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Professional README.
✅ It contains a home page.
✅ It contains the comments popup.
✅ The counters functions are covered with unit tests using [the Jest Framwework](https://jestjs.io/).


Cheers and Happy coding!👏👏👏

thats-how-its-done-allison-janney


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HammadSiddique** [comment]
@Cathella Thank you for the compliment and approving the project!

**Pull request merged**